src/components/Logo.js

```diff
@@ -1,17 +1,18 @@
-import Link from 'next/link'
 import React from 'react'
-
+import Link from 'next/link'
+import Image from 'next/image'
 
 
 
 const Logo = () => {
   return (
     
       <Link href="/" whileHover={{ scale: 1.5 }}>
-        <img
+        <Image
+          alt='This is a logo of Code with Aloha represented by the name, as well as an orange lightbulb with the torch like yellow flame akin to a sunset scene.'
```

Comment:
Example of reducing verbosity:
```suggestion
          alt='Code with Aloha'
```

This ensures that it serves the same functionality as it does for sighted users. The visual decorations aren't important to the functionality of the image.